java/com/google/flatbuffers/Table.java

```diff
@@ -69,7 +69,8 @@ protected int __offset(int vtable_offset) {
 
   protected static int __offset(int vtable_offset, int offset, ByteBuffer bb) {
     int vtable = bb.array().length - offset;
```

Comment:
Should `bb.array().length` not be `bb.limit()` to be totally correct? Just in case the backing array isn't fully used?